snapcraft/internal/build_providers/_multipass/_multipass.py

```diff
@@ -57,16 +57,10 @@ def _run(self, command, hide_output: bool = False) -> None:
         )
 
     def _get_disk_image(self) -> str:
-        if self.project.info.base == "core18":
-            image = "18.04"
-        elif self.project.info.base in ("core16", None):
-            image = "16.04"
+        if self.project.info.base is None:
+            image = "snapcraft:core16"
         else:
-            raise errors.UnsupportedHostError(
```

Comment:
Removing this doesn't quite provide as nice of an experience:

```
Launching a VM.
failed to launch: Unable to find an image matching "invalid"                    
An error occurred when trying to launch the instance with 'multipass': returned exit code 2.
Ensure that 'multipass' is setup correctly and try again.
```

Reply:
@kyrofa the message is nicer if you are on edge for multipass

Reply:
@sergiusens I am on edge. What message do you see?